kube/pod.go

```diff
@@ -110,11 +110,18 @@ func getContainerImageName(role *model.Role, settings *ExportSettings) (string,
 	if err != nil {
 		return "", err
 	}
-	imageName := builder.GetRoleDevImageName("", "", settings.Repository, role, devVersion)
-	value := "{{ default \"docker.io\" .Values.kube.registry -}}/"
-	value += "{{- .Values.kube.organization -}}/"
-	value += imageName
-	return value, nil
+
+	var imageName string
+	if settings.CreateHelmChart {
+		name := builder.GetRoleDevImageName("", "", settings.Repository, role, devVersion)
+		imageName = "{{ default \"docker.io\" .Values.kube.registry -}}/"
+		imageName += "{{- .Values.kube.organization -}}/"
```

Comment:
Not sure why you use `{{- ... -}}` instead of plain `{{ ... }}`. The dashes are eating up whitespace outside the template, but you don't have any whitespace there.

Reply:
Oh, I misread the docs. I thought it would trim whitespace from the variables being replaced in the template. You'd have to try pretty hard to break it in that way though.

Reply:
Yeah, to trim whitespace from the variable you would have to pass it through the `trim` template function: `{{ .Values.FOO | trim }}`

http://masterminds.github.io/sprig/strings.html